app/code/Magento/CmsGraphQl/Model/Resolver/Blocks.php

```diff
@@ -74,16 +77,16 @@ private function getBlockIdentifiers(array $args): array
      * Get blocks data
      *
      * @param array $blockIdentifiers
+     * @param Store $currentStore
      * @return array
-     * @throws GraphQlNoSuchEntityException
      */
-    private function getBlocksData(array $blockIdentifiers): array
+    private function getBlocksData(array $blockIdentifiers, Store $currentStore): array
     {
         $blocksData = [];
         foreach ($blockIdentifiers as $blockIdentifier) {
             try {
-                $blocksData[$blockIdentifier] = $this->blockDataProvider->getData($blockIdentifier);
-            } catch (NoSuchEntityException $e) {
+                $blocksData[$blockIdentifier] = $this->blockDataProvider->getData($blockIdentifier, $currentStore);
+            } catch (LocalizedException $e) {
```

Comment:
Why did you change to `LocalizedException`?

Reply:
Good question, I do not remember hahaha, changing it back